openmc/cell.py

```diff
@@ -23,7 +23,7 @@ def reset_auto_cell_id():
 
 
 class Cell(object):
-    """A region of space defined as the intersection of half-space created by
+    r"""A region of space defined as the intersection of half-space created by
```

Comment:
Exactly -- it's needed when there's math present because of all the backslashes. Otherwise they're interpreted as escape sequences.